metadata.yaml

```diff
@@ -9,11 +9,17 @@ docs: https://discourse.charmhub.io/t/mlflow-server-docs-index/10816
 containers:
   mlflow-server:
     resource: oci-image
+  mlflow-prometheus-exporter:
+    resource: exporter-oci-image
 resources:
   oci-image:
     type: oci-image
     description: Backing OCI image
     upstream-source: docker.io/charmedkubeflow/mlflow:v2.1.1_22.04_1
+  exporter-oci-image:
```

Comment:
The publish CI for this I think is failing because this is a new resource.  There are instructions [here](https://github.com/canonical/charming-actions/issues/58#issuecomment-1579282157)